test/redis_mutex/supervisor_test.exs

```diff
@@ -0,0 +1,39 @@
+defmodule RedisMutex.SupervisorTest do
+  use ExUnit.Case, async: true
+
+  import Mox
```

Comment:
you are using Mox rather than Hammox. Do you need a loose contract? Hammox is preferred if you want stricted. There is a difference between the 2

Reply:
I've removed the mocking entirely.

Given that this library is based on using Redis, is there a reason to have tests that _don't_ use Redis and require a double?